pywis_pubsub/cli_options.py

```diff
@@ -34,9 +34,9 @@ def OPTION_VERBOSITY(f):
     logging_options = ['ERROR', 'WARNING', 'INFO', 'DEBUG']
 
     def callback(ctx, param, value):
-        if value is not None:
-            logging.basicConfig(stream=sys.stdout,
-                                level=getattr(logging, value))
+        value2 = value or 'DEBUG'
```

Comment:
does this mean that if verbosity is not defined, the default is DEBUG ?

Reply:
Yes.  I adjusted the default to be `INFO`.  The driver was being able to see notifications on the CLI by default (current behaviour has no output once subscribed).  I've adjusted so INFO is the default logging level, and for topic and message payload to be emitted at the INFO level.